Json.More/JsonArrayTupleConverter.cs

```diff
@@ -48,26 +48,23 @@ public override ValueTuple<T> Read(ref Utf8JsonReader reader, Type typeToConvert
 	{
 		if (reader.TokenType != JsonTokenType.StartArray)
 			throw new JsonException("Expected tuple to be encoded as an array.");
-		reader.Read();
 
-		options = JsonCollectionTupleConverter.ManageConverters(options);
+		var array = JsonElement.ParseValue(ref reader);
```

Comment:
Can't comment on the exact line, but it goes without saying that this JsonConverterFactory won't work in Native AOT since it's using unsupported reflection (Type.MakeGenericType). Unfortunately, a source generator is needed to be able to fold a nested tuple converter in a safe way.

Reply:
Yeah, I'm still not sure how to get Native AOT working in .Net Standard 2.0.  There are other places I'd like to make things compatible.

Reply:
I also have other areas that use reflection, like the `EnumStringConverter`.